ci-operator/step-registry/openstack/provision/securitygroups/openstack-provision-securitygroups-commands.sh

```diff
@@ -31,6 +31,10 @@ if [[ -n "$ADDITIONAL_SECURITY_GROUP_RULES" ]]; then
 					echo "Adding ${service} rule to security group ${sg_id}" 
 					openstack security group rule create "$sg_id" --protocol tcp --dst-port 12865:12865 --remote-ip 0.0.0.0/0
 					;;
+				IPv6DualStack)
+					echo "Adding ${service} rule to security group ${sg_id}" 
+					openstack security group rule create "$sg_id" --protocol tcp --dst-port 9080:9080 --remote-ip 0.0.0.0/0
```

Comment:
why is this needed? Depending on the reason it might be worth to add a comment

Reply:
Hello, Don't worry. It was a test, I've removed it)